README.old

```diff
@@ -0,0 +1,24 @@
+# perseo
```

Comment:
No need of keeping the old readme. The new one is much better :)

Reply:
Ok. Fixed in 206a1246dba30113ff260e4d3057e6974a8a9299